otter_welcome_buddy/common/utils/dates.py

```diff
@@ -0,0 +1,17 @@
+from datetime import datetime
+
+from apscheduler.triggers.cron import CronTrigger
+
+
+class DateUtils:
+    """Utility class for datetime python library"""
+
+    @staticmethod
+    def get_current_month() -> int:
+        """Get current number month"""
+        return datetime.now().month
+
+    @staticmethod
+    def create_cron_trigger_from(crontab: str) -> CronTrigger:
```

Comment:
I would suggest changing this `crontab` string for an enum constant type or something similar where we add the cron expressions wanted example:
```python
# On contants.py
def CronExpressionsType(enum):
    DAY_ONE_OF_EACH_MONTH_CRON = "0 0 1 * *"

# On dates.py
def create_cron_trigger_from(crontab: CronExpressionsType) -> CronTrigger:
    ....
```